Makefile

```diff
@@ -44,3 +59,38 @@ format :
 	# show diff via black
 	black tests --diff
 	black stream2hop --diff
+
+print-%  : ; @echo $* = $($*)
+
+
+.PHONY: tns_container
+tns_container: 
+	@if [ ! -z "$$(git status --porcelain)" ]; then echo "Directory is not clean. Commit your changes."; exit 1; fi
+	docker build -f Dockerfile_TNS -t $(CNT_IMG) .
+	docker tag $(CNT_IMG) $(CNT_LTST)
+
+.PHONY: set-release-tags
+set-release-tags:
+	@$(eval RELEASE_TAG := $(shell echo $(GITHUB_REF) | awk -F- '{print $$2}'))
+	@echo RELEASE_TAG =  $(RELEASE_TAG)
+	@$(eval MAJOR_TAG   := $(shell echo $(RELEASE_TAG) | awk -F. '{print $$1}'))
+	@echo MAJOR_TAG = $(MAJOR_TAG)
+	@$(eval MINOR_TAG   := $(shell echo $(RELEASE_TAG) | awk -F. '{print $$2}'))
+	@echo MINOR_TAG = $(MINOR_TAG)
+
+.PHONY: docker_push
+docker_push: 
+	./scrips/awsDockerLogin $(REGION) $(AWSREG) >/dev/null 2>/dev/null
+	docker tag $(CNT_IMG) $(AWSREG)/$(CNT_NAME):$(RELEASE_TAG)
+	docker tag $(CNT_IMG) $(AWSREG)/$(CNT_NAME):$(MAJOR)
+	docker tag $(CNT_IMG) $(AWSREG)/$(CNT_NAME):$(MAJOR).$(MINOR)
+	docker push $(AWSREG)/$(CNT_NAME):$(RELEASE_TAG)
+	docker push $(AWSREG)/$(CNT_NAME):$(MAJOR)
+	docker push $(AWSREG)/$(CNT_NAME):$(MAJOR).$(MINOR)
+#	rm -f $(HOME)/.docker/config.json
```

Comment:
Remove this bit of cruft?

Reply:
I am not sure, the makefile changes are all copied from Ron's file. I will remove them.